common/commands/src/validator/mixnet/mod.rs

```diff
@@ -14,6 +14,7 @@ pub struct Mixnet {
     pub command: MixnetCommands,
 }
 
+#[allow(clippy::large_enum_variant)]
```

Comment:
should we be allowing this? or would it perhaps make sense to just Box the problematic variants?

Reply:
Yes, possibly. Though I this case I think the performance hit isn't an issue